cgmanifest.json

```diff
@@ -0,0 +1,34 @@
+{"Registrations":[ 
```

Comment:
hey @hlums are you sure this is needed (in this shape)?
In reco I did this: https://github.com/microsoft/recommenders/blob/staging/tests/ci/component_governance.yaml it generates the required files for component governance on the fly